src/meshes/t8code_mesh.jl

```diff
@@ -1092,6 +1255,21 @@ function fill_mesh_info!(mesh::T8codeMesh, interfaces, mortars, boundaries,
                 else
                     neighbor_level = t8_element_level(neighbor_scheme, neighbor_leaves[1])
 
+                    # Compute the `orientation` of the touching faces.
+                    if t8_element_is_root_boundary(eclass_scheme, element, iface) == 1
```

Comment:
```suggestion
                    if t8_element_is_root_boundary(eclass_scheme, element, iface)
```
Doesn't `t8_element_is_root_boundary` return a boolean? We have
```julia
julia> true == 1
true
```
but this looks strange to me if it's a boolean already

Reply:
When I do this as you suggest, I always get the error message:
```
ERROR: TypeError: non-boolean (Int64) used in boolean context
```

Reply:
Ah ok, so it does indeed return an integer and not a boolean, which is kind of strange in my opinion, but ok. Thanks!